app/src/main/res/values/strings.xml

```diff
@@ -265,10 +269,10 @@
     <!-- Feature discovery -->
     <string name="app_info_title">How The Algorithm Works</string>
     <string name="app_info_body">
-        1. Choose between two users repeatedly.\n
-        2. The app creates a stack ranking.\n
+        1. You choose between two users repeatedly.\n
```

Comment:
Character count is important with this one. The brief has a screenshot. Space is very limited. Plus, users won't take the time to read it. They'll scan it.

Ideally, you can decrease the word count/letter count.